platform/broadcom/sonic-platform-modules-dell/s6100/sonic_platform/chassis.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+#############################################################################
+# DELLEMC
+#
+# Module contains an implementation of SONiC Platform Base API and
+# provides the platform information
+#
+#############################################################################
+
+try:
+    import os
+    from sonic_platform_base.chassis_base import ChassisBase
+except ImportError as e:
+    raise ImportError(str(e) + "- required module not found")
+
+
+class Chassis(ChassisBase):
+    """
+    DELLEMC Platform-specific Chassis class
+    """
+
+    HWMON_DIR = "/sys/devices/platform/SMF.512/hwmon/"
+    HWMON_NODE = os.listdir(HWMON_DIR)[0]
+    MAILBOX_DIR = HWMON_DIR + HWMON_NODE
+
+    reset_reason_dict = {}
+    reset_reason_dict[11] = (ChassisBase.REBOOT_CAUSE_POWER_LOSS)
```

Comment:
Parenthesizing these here is unnecessary and a bit confusing since they are individual values, not tuples. I suggest removing the parentheses from around these values and changing lines 74-79 as follows to make it more clear:

```
        if (reset_reason == 11):
            if (power_reason in self.power_reason_dict):
                return (self.power_reason_dict[power_reason], None)
        else:
            if (reset_reason in self.reset_reason_dict):
                return (self.reset_reason_dict[reset_reason], None)
```

Reply:
Hi @jleveque 

Understood. Removed the paranthesis and returned None at relevant places..

thanks
Sridhar.R